src/sentry/api/endpoints/project_performance_issue_settings.py

```diff
@@ -17,18 +32,18 @@
 # These options should only be accessible internally and used by
 # support to enable/disable performance issue detection for an outlying project
 # on a case-by-case basis.
-internal_only_project_setting_options = [
-    "uncompressed_assets_detection_enabled",
-    "consecutive_http_spans_detection_enabled",
-    "large_http_payload_detection_enabled",
-    "n_plus_one_db_queries_detection_enabled",
-    "n_plus_one_api_calls_detection_enabled",
-    "db_on_main_thread_detection_enabled",
-    "file_io_on_main_thread_detection_enabled",
-    "consecutive_db_queries_detection_enabled",
-    "large_render_blocking_asset_detection_enabled",
-    "slow_db_queries_detection_enabled",
-]
+map_internal_only_project_settings_to_group: Dict[str, Type[GroupType]] = {
```

Comment:
```suggestion
internal_only_project_settings_to_group_map: Dict[str, Type[GroupType]] = {
```

This is low priority, and maybe you can just add it to a cleanup PR later but starting a name with `map` makes it look like it's an operation rather than a variable